### ISSUE TITLE
Write immediate successful cmor-fixer action to olist

### ISSUE THREAD

**Issue by treerink** [EC-Earth/cmor-fixer#15]
Write immediate successful `cmor-fixer` action to the `olist` during running  `cmor-fixer`, instead of writting the entire olist at the end just before  `cmor-fixer` ends (current situation). This is convenient in case of an interruption, so one can see to which point  `cmor-fixer` arrived.

**Comment by treerink** [comment]
While testing the `safe_olist` branch, I get the error:
```
INFO:cmor-fixer.py: Setting tracking_id to hdl:21.14100/16826e0b-4f8f-4c4e-a73f-4b89d1c5a24a for /nfs/home/users/reerink/l2/cmorised-results/cmor-cmip-test-all-11/t002/ifs/001/CMIP6/CMIP/EC-Earth-Consortium/EC-Earth3/piControl/r1i1p1f1/E3hrPt/ta/gr/v20191121/ta_E3hrPt_EC-Earth3_piControl_r1i1p1f1_gr_199001010000-199012312100.nc
INFO:cmor-fixer.py: Appending message about modification to the history attribute.
Traceback (most recent call last):
  File "./cmor-fixer.py", line 206, in <module>
    main()
  File "./cmor-fixer.py", line 180, in main
    worker(os.path.join(root, filepath))
  File "./cmor-fixer.py", line 97, in process_file
    if isinstance(flog, multiprocessing.Queue):
TypeError: isinstance() arg 2 must be a type or tuple of types
```

**Comment by treerink** [comment]
Thanks, after 893847fde1e56815e6a23855cfb5331aa56c02ef this issue is solved. I will continue testing.

**Comment by treerink** [comment]
Tests successful for all combinations of "dry-run" and "with and without interruptions".

**Comment by treerink** [comment]
The `safe_olist` branch is merged in #18.

When the `list-of-modified-files.txt` or a number-labeled version exists, it is not overwritten but a next number version is written. I changed it such that this also occurs when the `dry-run` mode is used. This makes the check in the instructions save.

Done, closing.
